### ISSUE TITLE
Component demo "Time Picker With Step Setting" throws NPE when clearing time value field

### ISSUE THREAD

**Issue by paulroemer** [vaadin/vaadin-time-picker-flow#94]
Where: 
- https://preview.vaadin.com/components/vaadin-time-picker/java-examples  (14.1.27)
- https://vaadin.com/components/vaadin-time-picker/java-examples (14.1.18)

What:
Select a time, delete it again and press enter.

```
vaadin.com(opens in new tab)APP Unhandled NullPointerException on page https://vaadin.com/components/vaadin-time-picker/java-examples

com.vaadin.flow.component.timepicker.demo.TimePickerView.lambda$createTimePickerWithStepSetting$1da64346$1(TimePickerView.java:150)
```

**Comment by Haprog** [comment]
The problem is this event listener that doesn't have a null check for the value: https://github.com/vaadin/vaadin-time-picker-flow/blob/2.0.6/vaadin-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/timepicker/demo/TimePickerView.java#L150

```java
        timePicker.addValueChangeListener(event -> {
            LocalTime value = event.getValue();
            localTimeValue.setText(String.format(localTimeValueFormat,
                    value.getHour(), value.getMinute(), value.getSecond(), // <- value is null when set to empty on client
                    value.get(ChronoField.MILLI_OF_SECOND)));
        });
```

Very easy to fix.

**Comment by web-padawan** [comment]
Fixed by #102
